openstack/nova/Chart.yaml

```diff
@@ -30,8 +30,8 @@ dependencies:
     repository: https://charts.eu-de-2.cloud.sap
     version: 0.4.2
   - name: utils
-    repository: https://charts.eu-de-2.cloud.sap
-    version: 0.5.0
+    repository: file://../utils
```

Comment:
I know this is still marked as draft, but we should not forget to update these references again after merging the PR we depend on, so I'm still marking it now.

Reply:
Has been updated.